packages/amplify-category-geo/src/commands/geo.ts

```diff
@@ -0,0 +1,43 @@
+import { $TSContext } from 'amplify-cli-core';
+import { category } from '../constants';
+
+export const name = category;
+
+export const run = async (context: $TSContext) => {
```

Comment:
I know this file naming is following the convention of other category plugins, but I think it makes more sense to put this in a file named `.../src/commands/geo/help.ts`